localTesting/run_tests_local.sh

```
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -euo pipefail
+
+# Загрузить .env
+export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)